.github/workflows/scripts/publish-sdkman.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -eu
+
+# latest stable dotty version 
+DOTTY_VERSION=$(curl -s https://api.github.com/repos/lampepfl/dotty/releases/latest  | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+DOTTY_URL="https://github.com/lampepfl/dotty/releases/download/$DOTTY_VERSION/scala3-$DOTTY_VERSION.zip"
```

Comment:
Can @marc0der confirm that this is the correct artifact to promote to SDKMAN?

Reply:
It looks sane to me. I tried those two lines in a terminal and the URL holds true :+1:

Reply:
Thank you! Can you also confirm that the content of the zip file is what we should send to SDKMAN? (sorry, I think my previous comment was not very clear about that)